main.tf

```diff
@@ -32,8 +35,9 @@ resource "aws_lambda_function" "lambda" {
   role             = "${aws_iam_role.lambda.arn}"
   handler          = "${var.handler}"
   source_code_hash = "${var.source_code_hash}"
-  count            = "${var.enabled}"
   timeout          = "${var.timeout}"
+
+  count = "${local.enabled}"
```

Comment:
Put `count` at the top of the arguments.

Reply:
Okay